Marlin/ultralcd.cpp

```diff
@@ -3527,59 +3522,64 @@ void lcd_update() {
         slow_buttons = lcd_implementation_read_slow_buttons(); // buttons which take too long to read in interrupt context
       #endif
 
-	  #if ENABLED(ADC_KEYPAD)
-      static int32_t adc_steps = 0;
-      if (buttons_reprapworld_keypad != 0)
-      {
-        adc_steps++;
-        if(adc_steps > 20)
-          adc_steps = 20;
-
-        lcd_quick_feedback();
-        lcdDrawUpdate = LCDVIEW_REDRAW_NOW;
-        return_to_status_ms = millis() + LCD_TIMEOUT_TO_STATUS;
-        if (encoderDirection == -1) // side effect which signals we are inside a menu
-        {
-          if (buttons_reprapworld_keypad&EN_REPRAPWORLD_KEYPAD_DOWN)
-            encoderPosition -= ENCODER_STEPS_PER_MENU_ITEM;
-          else if (buttons_reprapworld_keypad&EN_REPRAPWORLD_KEYPAD_UP)
-            encoderPosition += ENCODER_STEPS_PER_MENU_ITEM;
-          else if (buttons_reprapworld_keypad & EN_REPRAPWORLD_KEYPAD_LEFT)
-            menu_action_back();
-          else if (buttons_reprapworld_keypad&EN_REPRAPWORLD_KEYPAD_RIGHT)
-            // enqueue_and_echo_commands_P(PSTR("M0 Pause"));
-            lcd_return_to_status();
+      #if ENABLED(ADC_KEYPAD)
+        static int32_t adc_steps = 0;
+        if (buttons_reprapworld_keypad != 0) {
+          adc_steps++;
+          if(adc_steps > 20) {
+            adc_steps = 20;
+          }
+          lcd_quick_feedback();
+          lcdDrawUpdate = LCDVIEW_REDRAW_NOW;
+          return_to_status_ms = millis() + LCD_TIMEOUT_TO_STATUS;
+          if (encoderDirection == -1) { // side effect which signals we are inside a menu
+            if (buttons_reprapworld_keypad&EN_REPRAPWORLD_KEYPAD_DOWN) {
+              encoderPosition -= ENCODER_STEPS_PER_MENU_ITEM;
+            }
+            else if (buttons_reprapworld_keypad&EN_REPRAPWORLD_KEYPAD_UP) {
```

Comment:
i would let this to them. I always change styles per file and not just parts of the code. if just makes this PRs diff against my codebase bigger. Changing styles like this could also be automated. It is hard to do something like this on such a large codebase too. If you want to help them, do it on marlin dev itself after they approve how to do it.